README.md

```diff
@@ -1 +1 @@
-# trial-review-hello
\ No newline at end of file
+# Hello Microverse
```

Comment:
Please add a descriptive _**README.md**_ file to your project, you can use [this template](https://github.com/microverseinc/readme-template), this helps anyone who sees this project have a better understanding about the project and helps non-technical people also